x-pack/legacy/plugins/ml/public/application/settings/calendars/edit/new_calendar.test.js

```diff
@@ -127,7 +127,7 @@ describe('NewCalendar', () => {
   test('New event modal shown on New event button click', () => {
     const wrapper = mountWithIntl(<NewCalendar {...props} />);
 
-    const importButton = wrapper.find('[data-testid="ml_new_event"]');
+    const importButton = wrapper.find('[data-test-subj="ml_new_event"]');
```

Comment:
Just wondering, as these are JS files: Did you manually grep for `data-testid` to 'fix' that ?

Reply:
yes